bridge/client/app/_models/trace.ts

```diff
@@ -170,6 +170,15 @@ class Trace {
     return this.data.service;
   }
 
+  getChartLabel(): string {
+    return this.data.labels?.get("buildId") ?? this.time;
+    if(this.data.labels)
```

Comment:
this is never called after the `return` in the line above, is it?

Reply:
Yes, I messed up the suggestion in https://github.com/keptn/keptn/pull/1995#discussion_r449120486. It should have replaced the whole method body but looks like I didn't select it correctly.